.github/workflows/test.yml

```diff
@@ -64,9 +64,9 @@ jobs:
           - ubuntu: 20.04
             containerd: v1.5.13
           - ubuntu: 20.04
-            containerd: v1.6.8
+            containerd: v1.6.10
           - ubuntu: 22.04
-            containerd: v1.6.8
+            containerd: v1.6.10
```

Comment:
no need to test with `v1.7.0-beta.0` ?

Reply:
Substantially covered by the `main` branch.

Reply:
👌